source/_docs/automation/trigger.markdown

```diff
@@ -11,26 +11,42 @@ Triggers are what starts the processing of an {% term automation %} rule. When _
 An {% term automation %} can be triggered by an {% term event %}, a certain {% term entity %} {% term state %}, at a given time, and more. These can be specified directly or more flexible via templates. It is also possible to specify multiple triggers for one automation.
 
 - [Trigger ID](#trigger-id)
+  - [Video tutorial](#video-tutorial)
```

Comment:
This is not a TOC.

Seems like your editor auto-updated this list.

Reply:
It did, I wasn't sure if this was wanted or not.

Reply:
Ok, this is really annoying.  How do I use VSCode homeassistant.io dev contain and save this file without the TOC changes.  No matter what I do,  it adds them back on save.  This is a really poor feature built into dev container.

Reply:
it ended up being one of the recommended vscode extensions provided by dev container.  I'm not sure which one it was, I was frustrated and turned them all off in order to submit the TOC changes without it auto-adding everything back.  Super frustrating!

Reply:
It can be disabled, but on a lot of pages it is helpful.

In general, always selectively commit your changes and you should be good 👍